src/device-interface.d.ts

```diff
@@ -24,16 +24,23 @@ interface IdentityObject {
      emailAddress?: string,
 }
 
+interface InternalIdentityObject extends IdentityObject {
+     fullName?: string
+}
+
 interface CreditCardObject {
      id: string,
      title: string,
-     displaystring: string,
+     displayNumber: string,
      cardName?: string,
-     cardstring?: string,
      cardSecurityCode?: string,
      expirationMonth?: string,
      expirationYear?: string,
```

Comment:
Note that these are actually `number`s in the API. I've asked the native team to change the implementation to `string` and have a tiny patch locally to make sure the logic keeps working.

Reply:
normally I would say let's update the type to `string | number` in that case, because then Typescript can check that we don't accidentally assume it's one or the other (and cause exceptions with method calls etc)

but if you have this handled elsewhere and the change is in flight, then no change needed